install/0000_30_machine-api-operator_09_rbac.yaml

```diff
@@ -167,6 +167,17 @@ rules:
     - list
     - watch
 
+# the baremetal pod deployment uses hostNetwork, hostPort, and privileged
```

Comment:
Any reason to include it in the ClusterRole and not the Role? The latter worked fine for me

Reply:
I opted for the clusterRole as scc are cluster scoped resources.